src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -112,20 +116,69 @@ export const ARShowScreen = ({ navigation, route }) => {
   );
 };
 
+/**
+ * index creation function for model and texture
+ *
+ * @param {string} startDate           start date of the model
+ * @param {number} timePeriodInDays    time period in days of the model
+ * @param {array}  scenes              array of models
+ *
+ * @return {object} both parsed values as an object, like { modalIndex: 1, textureIndex: 1 }
+ */
+const multipleSceneIndexGenerator = async ({ startDate, timePeriodInDays, scenes }) =>
+  new Promise((resolve) => {
+    let modalIndex, textureIndex;
+
+    /* all models must have the same number of variable textures. Therefore, in order to obtain the 
+       number of textures, the textures of the first model were calculated. */
+    if (scenes.length > 1) {
+      const today = new Date(),
+        differenceDate = extendedMoment.range(startDate, today).diff('days'),
+        textureCount = scenes[0]?.localUris?.filter(
+          ({ type, stable }) => !stable && type === 'texture'
```

Comment:
why are you checking the type here? can there be different types we do not want to consider?

Reply:
To get the index number I need the number of variable textures a model has.
I don't think I need the other types.